### PR TITLE
Bump some images

### PR DESCRIPTION
**Pull request by zoidbergwill**
This is gonna conflict with and requires the jsonnet changes from https://github.com/buildbarn/bb-deployments/pull/13

This is similar to https://github.com/buildbarn/bb-deployments/commit/81abe8f7c899d43b1061bd70fc2c64a915d2937f, but bumps docker-compose and Kubernetes images

### CODE REVIEW

**Comment by Qinusty** [comment]
It should be noted that updating images may break unless config changes are also made. E.g. the scheduler config change made in https://github.com/buildbarn/bb-remote-execution/commit/e9cc231ea6359361f498b6a539b32e5f2c853634

**Comment by zoidbergwill** [comment]
I will test this today, hopefully

**Comment by zoidbergwill** [comment]
@Qinusty I made the changes you mentioned

**Comment by zoidbergwill** [comment]
This also partially solves https://github.com/buildbarn/bb-deployments/issues/10 ?

**Comment by zoidbergwill** [comment]
I also had to update the worker configmap:

```
            { name: 'container-image', value: 'docker://marketplace.gcr.io/google/rbe-ubuntu16-04@sha256:b4dad0bfc4951d619229ab15343a311f2415a16ef83bcaa55b44f4e2bf1cf635' },
```

File: update_deps/update_deps.go [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+	"log"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/google/go-containerregistry/pkg/crane"
+)
+
+func main() {
+	grepCmd := exec.Command("git", "grep", "-l", "image: ")
```
Comment (EdSchouten) [inline update_deps/update_deps.go]: I'm not a big fan of this kind of logic. What if Buildbarn gains graphics support, thereby adding an `image: ` configuration option to specify a picture? Would we want this code to also process such statements? Clearly not.

Please replace this by a hardcoded list of pathnames where substitutions need to take place.

File: update_deps/update_deps.go [hunk 2]
```diff
@@ -0,0 +1,103 @@
+package main
```
Comment (EdSchouten) [inline update_deps/update_deps.go]: Buildbarn repositories use `rules_go` to build Go binaries. Could you please add Bazel build infrastructure to this repository to be able to build and run this tool? I seem to remember @mickael-carl is also planning on adding some bits to this repository that need this. @mickael-carl, could you please work together with @zoidbergwill to ensure there is no race condition?

File: update_deps/update_deps.go (hunk 1 above)
Comment (zoidbergwill) [reply update_deps/update_deps.go]: We could also use _git-grep_'s _pathspec_ support doing something like `git grep -l "image: " -- "*.yml" "*.yaml"`, but using a hardcoded list will greatly simplify the code and make it more performant.

File: update_deps/update_deps.go (hunk 2 above)
Comment (zoidbergwill) [reply update_deps/update_deps.go]: Totally can do. Wasn't even sure this script was something you folks would be interested in maintaining. 😁

**Comment by finn-ball** [comment]
Would it be possible to remove the `update_deps.go` and perhaps just manually update them for now? I've just realised the latest buildbarn images don't work with the example Kubernetes configs.

**Comment by zoidbergwill** [comment]
@finn-ball Sure, can do

File: kubernetes/config/frontend.yaml [hunk 3]
```diff
@@ -13,7 +13,6 @@ data:
       schedulers: {
         'remote-execution': { address: 'scheduler:8982' },
       },
-      verifyActionResultCompleteness: true,
```
Comment (EdSchouten) [inline kubernetes/config/frontend.yaml:16]: This functionality should now be enabled by wrapping the AC configuration with a CompletenessCheckingBlobAccess.

**Comment by mickael-carl** [comment]
@zoidbergwill feel like rebasing this on top of master and update to latest images?

**Comment by zoidbergwill** [comment]
@mickael-carl I would've happily, but thanks for those that tried to help get this merged. 👍

**Pull request closed without merging**